crates/utils/src/config.rs

```diff
@@ -986,6 +986,13 @@ impl TlsConfig {
             client_key_path,
         }
     }
+
+    /// Whether the server tls is enabled
+    #[must_use]
+    #[inline]
+    pub fn server_tls_enabled(&self) -> bool {
+        self.server_cert_path.is_some() && self.server_key_path.is_some()
```

Comment:
"The server_cert seems a bit confusing, while the peer_cert is clearer. Just my two cents."

Reply:
Renamed